spec.md

```diff
@@ -2361,7 +2364,10 @@ circuit Foo:
     wire en: UInt<1>
     connect pred, eq(X, Y)
     connect en, Z_valid
-    assume(clk, pred, en, "X equals Y when Z is valid but got X=%d Y=%d", X, Y) : optional_name
+    assume(
+      clk, pred, en, "X equals Y when Z is valid but got X=%d Y=%d", X,
```

Comment:
While I'd rather the example just fit on one line, this is better than before.

Thanks!